calico/operations/troubleshoot/vpp.mdx

```diff
@@ -113,23 +112,23 @@ calivppctl vppctl myk8node1
 To list existing interfaces and basic counters use
 
 ```
-vpp show int
-vpp show int addr
+vpp# show int
```

Comment:
This looks like an odd change. Did the command name change to `vpp#`?

Reply:
"vpp#" is the cli prompt to VPP and I have just rectified the mistake where the "#" was missing :)